frontend/src/app/[locale]/(base)/organization/[id]/manage-users/actions.ts

```
@@ -70,14 +71,17 @@ export const inviteUserAction = async (
       requestData,
     );
 
-    return { data: inviteUserResponse, success: true };
+    return {
+      data: inviteUserResponse,
+      invitationCreated: inviteUserResponse.organization_invitation_id,
+    };
   } catch (e) {
     const error = e as Error;
     console.error(
       `Error inviting user to org - ${error.message} ${error.cause?.toString() || ""}`,
     );
     return {
-      errorMessage: error.message,
+      errorMessage: `${genericErrorMessage}: ${error.message}`,
```

I'd personally recommend just showing the generic error message and swallowing the internal one, but I leave that call up to design.